renku_sphinx_theme/static/custom.scss

```
@@ -1,3 +1,15 @@
+/**
+  CHANGE STYLE RELATED THINGS HERE!!!
+  
+  Requirements: 
+  - pip install compile-scss https://pypi.org/project/compile-scss/
+
```

I added `compile-scss` to the setup requirements, so this can be changed to 

```suggestion

To make changes to styling, first run the following in the root of the project:
    - pip install -e .[dev]
    
Then you can edit and make changes to this file and see the results by doing the following:
```